test/test_jit.py

```diff
@@ -4116,6 +4116,17 @@ def invalid_prefix_annotation3(a):
                     #     type: (Int) -> Int
                     return a + 2
 
+    def test_builtin_function_attributes(self):
```

Comment:
can we add a test to `test_type_sharing.py` to guard the type sharing behavior as well.